core/slim_globals.h

```diff
@@ -354,6 +354,432 @@ extern EidosValue_String_SP gStaticEidosValue_StringT;
 #define SLIM_TSK_INDIVIDUAL_REMEMBERED  ((uint32_t)(1 << 17))
 #define SLIM_TSK_INDIVIDUAL_FIRST_GEN   ((uint32_t)(1 << 18))
 
+// Metadata schemas:
+// These should be valid json strings, parseable by python's json.loads( )
+// and then turn into a valid metadata schema by tskit.MetadataSchema( ).
+// You can check these by doing, in python:
+// ```
+// t = ( <paste in everything below except final semicolon> )
+// d = json.loads(t)
+// m = tskit.MetadataSchema(d)
+// for e in d['examples']:
+//    m.encode_row(e)
+// ```
+// Furthermore, so that they match with the way python would do it,
+// we've produced these by doing :
+// ```
+// import pyslim, json
+// for ms in pyslim.slim_metadata_schemas:
+//   print(ms)
+//   print(str(pyslim.slim_metadata_schemas[ms]))
+// ```
+// See the pyslim code for readable versions of these.
+ 
+
+const std::string SLIM_TSK_METADATA_SCHEMA =
+"{\n"
```

Comment:
This got changed in tskit anyhow to remove all the linebreaks and spaces.

Reply:
OK.  I'm just saying for readability in the SLiM code here it would be much more readable as a multiline string literal...

Reply:
I've edited these darn things enough times by now I don't want to do that any more - for readability, can we just look at [the pyslim versions](https://github.com/tskit-dev/pyslim/blob/update_tskit/pyslim/slim_metadata.py#L18), since that's where I'm getting these from, anyhow?

Reply:
OK.  I feel your pain.